app/models/concerns/admin_collection_behavior.rb

```diff
@@ -18,15 +18,27 @@ def managers
     edit_users & collection_managers.to_a
   end
 
+  def inherited_managers
+    unit.unit_admins_and_managers
+  end
```

Comment:
We could maybe test these convenience methods, though that's probably redundant since I think the base methods are tested in the Admin::Unit suite...